include/open62541/server.h

```diff
@@ -90,6 +90,7 @@ struct UA_ServerConfig {
                     * then be retrieved e.g. in a callback that forwards a
                     * pointer to the server. */
     void (*displayNameChanged)(UA_Server *server, UA_NodeId *nodeId, UA_LocalizedText *newDisplayName);
+    void (*descriptionChanged)(UA_Server *server, UA_NodeId *nodeId, UA_LocalizedText *newDescription);
```

Comment:
You should strongly consider adding the void* context parameter to the callback. In fact both event the displayNameChanged could use it. There is a context parameter in the node itself which could be used for that. But we should think about if there is a case where node context is not the same as callback context.